public/js/main.js

```diff
@@ -185,9 +185,34 @@ filters.addEventListener("click",(event) => {
   const filterSelected = document.getElementsByClassName("selected")[0];
   filterSelected.removeAttribute("class");
   element.className = "selected";
-  renderData();
+  todoListComputed();
 });
 
+listData.addEventListener("dblclick",(event) => {
+  const element = event.target;
+  const item = element.parentNode.parentNode;
+  item.classList.add("editing");
+  const inputEdit = item.lastElementChild;
+
+  inputEdit.onkeypress = (e) => {
+    const enterKey = 13;
```

Comment:
`enterKey` variable duplicated in many functions, should put it out of the function
and should capitalize this one